sources/north-america/us/tx/Smith_County_Orthophoto_2019_WMS.geojson

```diff
@@ -4,10 +4,10 @@
         "license_url": "https://data.tnris.org/collection/38f1b2e7-6fc7-4357-80a4-1ee33d7abaaf",
         "id": "smith_county_2019_wms",
         "type": "wms",
-        "name": "Smith County Imagery 2019",
+        "name": "TX: Smith County Imagery 2020",
```

Comment:
https://data.tnris.org/collection/38f1b2e7-6fc7-4357-80a4-1ee33d7abaaf names the dataset  "Smith County Imagery 2019", but this could be an error on this site. In the description it says `The Smith County GIS Consortium consists of the Smith County 911 Communications District, City of Tyler, Smith County Appraisal District, and Smith County. This 6-inch resolution leaf-off aerial imagery was acquired in three missions all flown on December 11, 2019. `

Reply:
I noticed that, too. The data was taken in 2019, but wasn't posted until 2020. I went with the posting date instead of the collection date, since it was within a month of 2020. I'll change it to better reflect collection date.